<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>783</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1360</width>
    <height>783</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1360</width>
    <height>783</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>二叉树</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
background-color: rgb(255, 255, 255);
	border-image: url(:/backpage_common.png);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="rbt_pre">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>前序遍历</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbt_post">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>后序遍历</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_in">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>771</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0.5px groove gray;;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,128);</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_prexpost">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>771</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0.5px groove gray;;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,128);</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_in">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>中序</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_x">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>前序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_draw">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>30</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_draw{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 255);
}

QPushButton#btn_draw:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,255);
}
 
QPushButton#btn_draw:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 255);
padding-left:6px;
padding-top:6px;
}

QPushButton#btn_draw:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exm">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>30</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_exm{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 255);
}

QPushButton#btn_exm:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,255);
}
 
QPushButton#btn_exm:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 255);
padding-left:6px;
padding-top:6px;
}

QPushButton#btn_exm:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
    </property>
    <property name="text">
     <string>Exm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_testPic">
    <property name="geometry">
     <rect>
      <x>1108</x>
      <y>609</y>
      <width>252</width>
      <height>174</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>130</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_clear{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 255);
}

QPushButton#btn_clear:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,255);
}
 
QPushButton#btn_clear:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 255);
padding-left:6px;
padding-top:6px;
}

QPushButton#btn_clear:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
    </property>
    <property name="text">
     <string>归归归归归零</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbb_travelWay">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>220</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;

    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
}
QComboBox::down-arrow {
	image: url(:/arrow_down.png);
}


QComboBox#cbb_travelWay{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgba(255,255,255,255);
}

QComboBox#cbb_travelWay:hover{
border-color: rgb(213, 198, 255);
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255,255);
}</string>
    </property>
    <item>
     <property name="text">
      <string>前序</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>中序</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>后序</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>层次</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_exmAll">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>220</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_exmAll{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(255,255,255, 255);
}

QPushButton#btn_exmAll:hover{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(240,255,255,255);
}
 
QPushButton#btn_exmAll:pressed{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(0, 0, 0);
background-color: rgba(225,255,255, 255);
padding-left:6px;
padding-top:6px;
}

QPushButton#btn_exmAll:disabled{
border:1px groove gray;
border-radius:5px;
padding:2px 4px;
color: rgb(176,196,222);
background-color: rgba(255,255,255,255);
padding-left:6px;
padding-top:6px;
}</string>
    </property>
    <property name="text">
     <string>All For One</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
